README.md

```diff
@@ -68,10 +68,15 @@ $ yarn install
 # serve with hot reload at localhost:3000
 $ yarn dev
 
-# build for production and launch server
+# server with hot reload and server-side rendering at localhost:3000
+$ yarn dev:ssr
+
+# build for production
+# you will need a web server to host the client
 $ yarn build
-$ yarn start
 
-# generate static project
-$ yarn generate
+# build for production with server-side rendering and launch server
+$ yarn build:ssr
```

Comment:
I'd detail this part saying that we can copy the `.nuxt` and `static` dirs somewhere else and run the `yarn install` with the custom package.json file to save space